src/bors/handlers/workflow.rs

```diff
@@ -235,6 +237,31 @@ async fn complete_try_build(
     Ok(())
 }
 
+/// Complete auto build workflow.
+async fn complete_auto_build(
+    repo: &RepositoryState,
+    pr: PullRequestModel,
+    workflows: Vec<WorkflowModel>,
+    has_failure: bool,
+    payload: CheckSuiteCompleted,
+) -> anyhow::Result<()> {
+    let comment = if !has_failure {
+        tracing::info!("Auto build succeeded for PR {}", pr.number);
+        auto_build_succeeded_comment(
```

Comment:
We shouldn't post the comment here, when the actual merge happens elsewhere. There could be a long delay between the comment and the actual merge, which is confusing. We should post a message when a failure happens, but if the build was successful, we should only trigger the merge queue handler, and let it do the merge and send the comment at the ~same time.